src/Swr/swrMultiplayer.c

```diff
@@ -3,8 +3,8 @@
 #include "globals.h"
 #include "macros.h"
 
-#include <stdComm.h>
-#include <sithMulti.h>
+#include <Win95/stdComm.h>
+#include <Dss/sithMulti.h>
 
```

Comment:
@tim-tim707 is the intention to go to this lack of path style on includes going forward? I see the changes you made to compile.bat, but most files still use <Win95/header.h> style instead

I have an alternative fix redoing the cmakelists, but went for this first